# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fetchmail/heimdal_compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -urN fetchmail-6.2.5.orig/kerberos.h fetchmail-6.2.5/kerberos.h
--- fetchmail-6.2.5.orig/kerberos.h	2003-07-17 03:03:23.000000000 +0200
+++ fetchmail-6.2.5/kerberos.h	2005-08-31 14:28:32.281090750 +0200
@@ -11,7 +11,6 @@
 
 #ifdef KERBEROS_V5
 #include <krb5.h>
-#include <com_err.h>
 #endif
 
 #ifdef KERBEROS_V4
